test/cache/test_cosmos_db_cache.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3 -m pytest
+
+import pickle
+import unittest
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+try:
+    from autogen.cache.cosmos_db_cache import CosmosDBCache
+
+    skip_cosmos_tests = False
+except ImportError:
+    skip_cosmos_tests = True
+
+
+class TestCosmosDBCache(unittest.TestCase):
```

Comment:
These test are not run in CI. Could you check build.yaml and add the extra option needed in the same section where redis options are installed?